### ISSUE TITLE
App does not load all assets sometimes and has to be refreshed

### ISSUE THREAD

**Issue by ahadshams** [webaverse/app#2114]
Many times the app loads without avatar and sometimes even broken UI

![image](https://user-images.githubusercontent.com/44798013/147743817-90cd1fc9-9b6a-4a1c-ad7b-62562ccaa132.png)

**Comment by ahadshams** [comment]
![image](https://user-images.githubusercontent.com/44798013/147743895-6c28d695-07fe-410c-b586-a9887c776743.png)

**Comment by abeersaqib** [comment]
![image](https://user-images.githubusercontent.com/5277220/147744464-7c22b221-3220-487f-9e87-1221bc74c694.png)

TransferGizmo is causing a failure

**Comment by avaer** [comment]
> TransferGizmo is causing a failure

No, that is not related to anything. It is an event race with the mouse and harmless.

**Comment by avaer** [comment]
@ahadshams this is a poor bug report.

Is this local? Some hosted? It looks like a GL error so what is the hardware, etc.

**Comment by avaer** [comment]
No response from the filers

**Comment by ahadshams** [comment]
I have re opened the issue and added more details to the issue description above
